.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Unit Tests
+
+on: [pull_request]
+
+jobs:
+  matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      latest: ${{ steps.set-matrix.outputs.requireds }}
+    steps:
+      - uses: ljharb/actions/node/matrix@main
+        id: set-matrix
+        with:
+          versionsAsRoot: true
+          type: majors
+          preset: '>=12'
```

Comment:
```suggestion
          preset: '^12 || ^14 || >= 16'
```

Reply:
Oh, slick. Updated that and removed the `excludes` list.